modules/kg-schemas/src/main/resources/schemas/nexus/core/datadownload/v0.1.0.json

```diff
@@ -0,0 +1,146 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    {
+      "nxcv": "https://bbp-nexus.epfl.ch/vocabs/nexus/core/v0.0.1/",
+      "sqvsh": "{{base}}/schemas/nexus/schemaorg/quantitativevalue/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/nexus/core/datadownload/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "owl:Ontology",
+  "imports": [
+    "{{base}}/schemas/nexus/schemaorg/quantitativevalue/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:DataDownloadShape",
+      "@type": "sh:NodeShape",
+      "label": "",
+      "targetClass": "nxcv:DataDownload",
+      "property": [
+        {
+          "path": "nxcv:contentSize",
+          "node": "this:ContentSizeQuantitativeValueShape",
+          "maxCount": 1
+        },
+        {
+          "path": "nxcv:digest",
+          "node": "this:DigestShape",
+          "maxCount": 1
+        },
+        {
+          "or": [
+            {
+              "path": "nxcv:downloadURL",
+              "datatype": "xsd:anyURI",
```

Comment:
`xsd:anyURI` allows both absolute or relative urls, could we further constrain it to an absolute URL?